platformio.ini

```diff
@@ -19,8 +19,8 @@ board_build.filesystem = littlefs
 extra_scripts = replace_fs.py
 upload_speed = 921600
 monitor_speed = 115200
-upload_port = /dev/ttyUSB0
```

Comment:
I think maybe we can get rid of `monitor_port` and `upload_port` here altogether, since they're gonna be different for different people anyways

Reply:
Yep, I agree with that! If it's possible to leave that out of the ini file, I think that'd be the way to go.